frontend/src/components/CredentialTile.vue

```diff
@@ -0,0 +1,143 @@
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <v-list-item :dense="dense">
+    <v-list-item-icon>
+      <v-icon v-if="!dense" :color="iconColor">mdi-key-change</v-icon>
+    </v-list-item-icon>
+    <v-list-item-content :class="{'py-0 my-0' : dense}">
+      <v-list-item-title class="d-flex align-center">
+        {{title}}
+        <v-tooltip top :disabled="!showChipTooltip">
+          <template v-slot:activator="{ on }">
+            <v-chip v-on="on" v-if="showChip" :color="phaseColor" label x-small class="ml-2" outlined>{{phaseCaption}}</v-chip>
+          </template>
+          All two-phase operations need to be in phase "Prepared" in order to complete the rotation of all credentials
```

Comment:
If I see 2/3 are prepared, how would I know which operation is the last one that needs to be prepared?
Shouldn't we add this information also in the tooltip?

Reply:
I added the information to the tooltip